web/themes/custom/fluffiness/templates/node--hospital.html.twig

```diff
@@ -77,9 +77,10 @@
   node.isSticky() ? 'node--sticky',
   not node.isPublished() ? 'node--unpublished',
   view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  'hospital--color-' ~ testvalue|lower|replace({'#': "-"}),
```

Comment:
Being a bit picky here 😬  class should better be `hospital__color-`

Reply:
also this should be a conditional, so we should check if `testvalue` is not empty/exists